peer-test-suite/docker/Dockerfile

```
@@ -1,6 +1,15 @@
+
 FROM node:14
-USER root
-RUN apt-get update && apt-get install -y sudo
-COPY . /app
+
+# Simple Error: Misspelled Dockerfile instruction (Case insensitive but 'RUNN' is invalid)
+RUNN apt-get update
```

Hadolint (2.14.0)

[error] missing whitespace

(DL1000)